### PR TITLE
Upgrade: Update project dependencies

### PR DESCRIPTION
**Pull request by ingmarh**
Also fixes vulnerabilities found via `npm audit`.

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -104,12 +104,13 @@ class App extends React.Component {
     )
   }
 
-  debugOnChange = (newValue) => {
+  handleChange = (newValue) => {
```
Comment (JonathanWbn) [inline public/index.js:107]: Did eslint ask you to do this?

File: test/form/Form.spec.jsx [hunk 2]
```diff
@@ -85,8 +85,7 @@ function render (props) {
           <TestField name="address.street" className="street" />
           <div>{`pristine ${pristine}`}</div>
           <div>{`submitting ${submitting}`}</div>
-        </div>
-      }
+        </div>}
```
Comment (JonathanWbn) [inline test/form/Form.spec.jsx]: Wow. Good thing we don't have to discuss formatting anymore, 'cause I would be fighting against this to my death. 😆

**Review by JonathanWbn — DISMISSED** [review]

File: public/index.js (hunk 1 above)
Comment (ingmarh) [reply public/index.js:107]: Yes. It's `react/jsx-handler-names` of `eslint-plugin-react`.

File: test/form/Form.spec.jsx (hunk 2 above)
Comment (ingmarh) [reply test/form/Form.spec.jsx]: Oh, this looks like a formatting "error" to me (it actually was auto-formatted via `react/jsx-curly-newline`).
I dislike it so much that I added braces now (force-pushed; see [the diff](https://github.com/ePages-de/react-components/compare/543b6418f998f0a948b64efeb4843fcd53e6dd42..43b194527b493d371c28221a22f2337ec065c705)).

**Review by JonathanWbn — APPROVED** [review]

**Pull request merged**